docs/madr/decisions/049-policy-on-namespace.md

```diff
@@ -0,0 +1,681 @@
+  to:
+    - targetRef:
+        kind: MeshHTTPRoute # doesn't work like this today, we need https://github.com/kumahq/kuma/issues/10247
+        name: backend-route
+      default:
+        http:
+          requestTimeout: 5s
+```
+
+```yaml
+# zone-2, consumer policy
+apiVersion: kuma.io/v1alpha1
+kind: MeshTimeout
+metadata:
+  name: frontend-to-backend
+  namespace: frontend-ns
+  labels:
+    kuma.io/zone: zone-2
+spec:
+  to:
+    - targetRef:
+        kind: MeshHTTPRoute # technically we can use 'name' and 'namespace' but we'd have to know hashed name of the route after syncing
+        labels:
+          kuma.io/display-name: backend-route
+          k8s.kuma.io/namespace: backend-ns
+          kuma.io/zone: zone-1
+      default:
+        http:
+          requestTimeout: 3s
+```
+
+### Referencing MeshGateway
+
+`MeshGateway` at this moment is not namespace-scoped. It can change in the future with
+the [MeshBuiltinGateway resource](https://github.com/kumahq/kuma/issues/10014).
+
+For policies in custom namespaces, we've decided that we would allow targeting `MeshGateway` in topLevel targetRef in 
+`MeshHTTPRoute` and `MeshTCPRoute`, and policies that configures only Envoy clusters (policies that allows configuring: `to[].targetRef.kind: MeshService` like `MeshCircuitBreaker`). 
+For other policies we would allow referencing Mesh*Routes that reference `MeshGateway` in topLevel targetRef.
+
+#### Examples
+
+As a **Cluster Operator** I want to deploy a single global `MeshGateway` to the cluster so that I’m able to manage a
+single deployment for the whole cluster without knowing specific details about apps and routes.
+
+```yaml
+apiVersion: kuma.io/v1alpha1
+kind: MeshGateway
+mesh: default
+metadata:
+  name: edge-gateway
+  labels:
+    kuma.io/origin: zone
+spec:
+  conf:
+    listeners:
+      - port: 80
+        protocol: HTTP
+  selectors:
+    - match:
+        kuma.io/service: edge-gateway
+```
+
+##### Targeting MeshGateway in MeshHttpRoute
+
+As a **Frontend Service Owner** I want to configure routes on `MeshGateway` without the access to
+cluster-scoped CRDs so that I could manage my routes independently.
+
+```yaml
+apiVersion: kuma.io/v1alpha1
+kind: MeshHTTPRoute
+metadata:
+  name: http-route-1
+  namespace: frontend-ns
+  labels:
+    kuma.io/origin: zone-1
+spec:
+  targetRef:
+    kind: MeshGateway
+    name: edge-gateway
+  to:
+    - targetRef:
+        kind: Mesh
+      rules:
+        - matches:
+            - path:
+                type: PathPrefix
+                value: /gui
+          default:
+            backendRefs:
+              - kind: MeshService
+                name: frontend
+                # namespace: frontend-ns - is added automatically
+```
+
+##### Targeting MeshGateway in other policies
+
+###### Producer policies
+
+As a **Frontend Service Owner** I want to configure circuit breaker on `MeshGateway` without the access to
+cluster-scoped CRDs so that I could manage my routes independently.
```

Comment:
This equivalent to a specific MeshSubset right?

Reply:
can you elaborate? I don't think I understand the comment?

Reply:
A MeshGateway is very similar to a MeshSubset no? In the end it just identifies a subset of all dataplanes

Reply:
That's right, I've added this info to the MADR